src/helix-ui/elements/HXMenuElement.js

```diff
@@ -1,193 +1,33 @@
 import { HXElement } from './HXElement';
-import { getPosition } from '../utils/position';
-import debounce from 'lodash/debounce';
 
-const DEFAULT_POSITION = 'bottom-start';
+import { mix } from '../utils';
+import { Positionable } from '../mixins/Positionable';
 
-/**
- * Fires when the element is concealed.
- *
- * @event Menu:close
- * @since 0.6.0
- * @type {CustomEvent}
- */
-
-/**
- * Fires when the element is revealed.
- *
- * @event Menu:open
- * @since 0.6.0
- * @type {CustomEvent}
- */
+class _ProtoClass extends mix(HXElement, Positionable) {}
 
 /**
  * Defines behavior for the `<hx-menu>` element.
  *
  * @emits Menu:close
  * @emits Menu:open
  * @extends HXElement
+ * @extends Positionable
  * @hideconstructor
  * @since 0.2.0
  */
-export class HXMenuElement extends HXElement {
+export class HXMenuElement extends _ProtoClass {
     static get is () {
         return 'hx-menu';
     }
 
+    /** @override */
     $onCreate () {
-        this._onDocumentClick = this._onDocumentClick.bind(this);
-        this._onDocumentScroll = this._onDocumentScroll.bind(this);
-        this._reposition = this._reposition.bind(this);
-
-        this._onWindowResize = debounce(this._reposition, 50);
-    }
-
-    $onConnect () {
-        this.$upgradeProperty('open');
-        this.$upgradeProperty('position');
-        this.$upgradeProperty('relativeTo');
-
-        this.$defaultAttribute('position', DEFAULT_POSITION);
-        this.$defaultAttribute('role', 'menu');
-
-        this.setAttribute('aria-hidden', !this.open);
-        this.setAttribute('aria-expanded', this.open);
```

Comment:
same -- is this supposed to be inside `$onCreate () {...}` or `$onConnect(){...}`?  just wanted to double check because i see it inside `$onCreate () {...}` -- just wanted to double check with the above comment that @CITguy left

Reply:
same -- should this be in $onConnect() or $onCreate()?